maths/cube_root.py

```diff
@@ -0,0 +1,10 @@
+
+def cube_root(a: float) ->float:
```

Comment:
```suggestion
def cube_root(a: float) -> float:
```
This will fix a PEP 8 error